packs/equipment/tacticians-helm.json

```diff
@@ -9,7 +9,7 @@
         },
         "containerId": null,
         "description": {
-            "value": "<p>Repurposing and enchanting a helmet worn by a battlefield commander can create a tactician's helm, imparting knowledge of battlefield tactics that feeds off your minor victories. The helm grants you a +1 item bonus to Warfare Lore checks. Also, a jewel adorns the brow of the helmet. This jewel becomes charged each time you hit a creature with a @UUID[Compendium.pf2e.actionspf2e.Item.Reactive Strike]. A tactician's helm can hold up to 2 charges, and its charges reset to 0 when you invest it.</p>\n<p><strong>Activate</strong> <span class=\"action-glyph\">1</span> command</p>\n<p><strong>Requirements</strong> The helm's jewel is charged</p>\n<p><strong>Frequency</strong> once per hour</p>\n<hr />\n<p><strong>Effect</strong> One charge in the helm's jewel expires, and you choose one of the following effects.</p>\n<ul>\n<li><strong>Charge!</strong> Stride twice.</li>\n<li><strong>Move It!</strong> You gain a +2 status bonus to Acrobatics and Athletics checks until the end of this turn. @UUID[Compendium.pf2e.equipment-effects.Item.Effect: Tactician's Helm (Move It!)]</li>\n<li><strong>Protect!</strong> If you're wielding a shield, Stride to a space adjacent to an ally, then Raise your Shield.</li>\n<li><strong>Re-Arm!</strong> Interact up to three times. Each of these actions must be used to do something on @UUID[Compendium.pf2e.journals.JournalEntry.S55aqwWIzpQRFhcq.JournalEntryPage.WL8YX3zIl4AhIrN9]{Changing Equipment}.</li>\n</ul>"
+            "value": "<p>Repurposing and enchanting a helmet worn by a battlefield commander can create a tactician's helm, imparting knowledge of battlefield tactics that feeds off your minor victories. The helm grants you a +1 item bonus to Warfare Lore checks. Also, a jewel adorns the brow of the helmet. This jewel becomes charged each time you hit a creature with a @UUID[Compendium.pf2e.actionspf2e.Item.Reactive Strike]. A tactician's helm can hold up to 2 charges, and its charges reset to 0 when you invest it.</p>\n<p><strong>Activate</strong> <span class=\"action-glyph\">1</span> command</p>\n<p><strong>Requirements</strong> The helm's jewel is charged</p>\n<p><strong>Frequency</strong> once per hour</p><hr /><p><strong>Effect</strong> One charge in the helm's jewel expires, and you choose one of the following effects.</p><ul><li><strong>Charge!</strong> Stride twice.</li><li><strong>Move It!</strong> You gain a +2 status bonus to Acrobatics and Athletics checks until the end of this turn. @UUID[Compendium.pf2e.equipment-effects.Item.Effect: Tactician's Helm (Move It!)]</li><li><strong>Protect!</strong> If you're wielding a shield, Stride to a space adjacent to an ally, then Raise your Shield.</li><li><strong>Re-Arm!</strong> Interact up to three times. Each of these actions must be used to do something on Changing Equipment.</li></ul>"
```

Comment:
I guess we no longer have an equivalent journal entry? PC p. 268 lists equipment actions.

Reply:
That would be correct - had to trim some fat out of the GM screen. This one was one of the casualties :|